packages/relay/tests/lib/eth.spec.ts

```diff
@@ -3609,7 +3609,7 @@ describe('Eth', async function () {
   let ethImpl: EthImpl;
   this.beforeAll(() => {
     // @ts-ignore
-    ethImpl = new EthImpl(null, mirrorNodeInstance, logger);
+    ethImpl = new EthImpl(null, mirrorNodeInstance, logger, '0x12a', new Registry());
```

Comment:
The other instantiation of `EthImpl` in this file (L113) supplies a cache argument; this one doesn't.  Why does one need the cache and the other not?  (Does the cache help the test up there?)

Reply:
The  cache is always used, is just that if you don't inject the cache, the constructor is going to instantiate one instance only for this class.

<img width="645" alt="image" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/b937b49b-008f-485f-920d-829a90b990a7">